react/week2/todolist/src/App.js

```diff
@@ -0,0 +1,76 @@
+import React, { useState } from 'react';
+import ReactDOM from 'react-dom';
+import logo from './logo.svg';
+import './App.css';
+
+let tasks = ["submit hyf homework", 
+            "go for a run", 
+            "watach movie"];
+
+            
+function TodoTimer(props){
+  const [count, setCount] = useState(0);
+  setTimeout(() => setCount(count + 1), 1000);
```

Comment:
Could this be implemented with the useEffect hook as in Younes' example?

Reply:
yes, this week I learned about this hook so implemented now.

Reply:
Thanks for your comments with good description. If something looks wrong in code please comment.